build.gradle.kts

```diff
@@ -104,7 +101,10 @@ dependencies {
     api("sc.fiji:bigdataviewer-vistools:1.0.0-beta-28")
 
     //TODO revert to official BVV
-    api("graphics.scenery:bigvolumeviewer:7698a01")
+    api("graphics.scenery:bigvolumeviewer:7698a01") {
+        exclude("org.jogamp.gluegen", "gluegen-rt")
+        exclude("org.jogamp.jogl", "jogl-all")
```

Comment:
These should be `gluegen-rt-main` and `jogl-all-main` respectively. I tested it, and as written the deps still come in, but if you exclude the main artifacts then they are successfully suppressed.